### PR TITLE
Contributor Guide edits

### PR DESCRIPTION
**Pull request by ntache-81**
First round edits to Contributor Guide.

### CODE REVIEW

File: contributions.md [hunk 1]
```diff
@@ -15,22 +15,34 @@ We are looking for contributions in the following areas:
...
-# Title Of Practice
+# Title of Practice
 
-A 2-3 sentence description of the practice followed by a sentence or two describing the main benefits of adopting this practice. Try to keep it brief.
+A 2-3 sentence description of the practice, followed by a sentence or two describing the main benefits of adopting this practice. Try to keep the text brief, relatable, and motivational.
 
 ## Nuance
 
-This section outlines 2-4 common pitfalls, challenges, or limitations teams commonly encounter when adopting this practice. Each nuance should have a subheader and description.
+This next section outlines 2-4 common pitfalls, challenges, or limitations that teams commonly encounter when adopting this practice. Each nuance should have a title (styled as a subhead) and a brief (3-5 sentences) description.
```
Comment (ntache-81) [inline contributions.md:27]: I did a quick check of practices edited thus far: 
- Funtional Core, Imperative Shell has 6 nuances
- Reduce Coupling has 6 nuances
- Run Pair Programming has 4 nuances

Edit first line here to: “This next section outlines ideally 3-4 common pitfalls…” ?

File: contributions.md [hunk 2]
```diff
@@ -15,22 +15,34 @@ We are looking for contributions in the following areas:
...
+
+*Begin this section with the following text:* "The following actions will help your team implement this practice." 
+
+Then, list the relevant general practices - from [Learning Culture](/capabilities/learning-culture.md) - and how to put those general practices to use by adding specifics such as talking points, demonstration instructions, roundtable discussion prompts, and links to external resources. 
+
+For example, a general practice is to Facilitate a Roundtable Discussion. In the Run Pair Programming Sessions practice, we elaborate on how to facilitate a roundtable discussion by listing specific discussion prompts related to pair programming such as, "How frequently do we engage in pair programming sessions, and are they integrated into our regular workflow?" 
+
+Another example of a general practice is to Start a Book Club. In the Reduce Coupling Between Abstractions practice, we list specific books that can help put this practice to use such as "Refactoring" by Martin Fowler and "Clean Architecture" by Robert C. Martin.
+
+See the relevant section in [other practice pages](/practices/) for more examples.
```
Comment (ntache-81) [inline contributions.md:41]: I'll edit this once we land on the name of this section. ;)

File: contributions.md [hunk 3]
```diff
@@ -15,22 +15,34 @@ We are looking for contributions in the following areas:
...
 ## Nuance
 
-This section outlines 2-4 common pitfalls, challenges, or limitations teams commonly encounter when adopting this practice. Each nuance should have a subheader and description.
+This next section outlines 2-4 common pitfalls, challenges, or limitations that teams commonly encounter when adopting this practice. Each nuance should have a title (styled as a subhead) and a brief (3-5 sentences) description.
 
-## How To Improve
+*Begin this section with the following text:* "This section outlines common pitfalls, challenges, or limitations teams commonly encounter when applying this practice. The goal here is not to discourage you. Rather, the goal is to arm you with the appropriate context so that you can make an informed decision about when and how to implement the practice with your team(s)."
 
-List practices from [Learning Culture](/capabilities/learning-culture.md) and add the practice specific information (talking points, instructions, links to external resources, etc) in the description. See the How To Improve sections in [other practice pages](/practices/) for examples.
+## Putting the Practice to Use [or "Practice in Action"?]
```
Comment (ntache-81) [inline contributions.md:31]: Thoughts on new title? In my mind, this is how we're bridging the content -- we're taking what we introduce as a general practice (like book club) and then listing specifics that put that general practice to use to implement a specific practice (like pair programming).

File: contributions.md [hunk 4]
```diff
@@ -15,22 +15,34 @@ We are looking for contributions in the following areas:
...
+
+*Begin this section with the following text:* "The following actions will help your team implement this practice."
```
Comment (ntache-81) [inline contributions.md:33]: OK? Other ideas for the boilerplate text: 
    - “This section elaborates on general practices, giving you specific and actionable ways to put this practice to use.” 
    - “This section provides specific and actionable ideas for putting this practice to use.”

File: contributions.md [hunk 5]
```diff
@@ -15,22 +15,34 @@ We are looking for contributions in the following areas:
...
+## Putting the Practice to Use [or "Practice in Action"?]
+
+*Begin this section with the following text:* "The following actions will help your team implement this practice." 
+
+Then, list the relevant general practices - from [Learning Culture](/capabilities/learning-culture.md) - and how to put those general practices to use by adding specifics such as talking points, demonstration instructions, roundtable discussion prompts, and links to external resources. 
+
+For example, a general practice is to Facilitate a Roundtable Discussion. In the Run Pair Programming Sessions practice, we elaborate on how to facilitate a roundtable discussion by listing specific discussion prompts related to pair programming such as, "How frequently do we engage in pair programming sessions, and are they integrated into our regular workflow?" 
+
+Another example of a general practice is to Start a Book Club. In the Reduce Coupling Between Abstractions practice, we list specific books that can help put this practice to use such as "Refactoring" by Martin Fowler and "Clean Architecture" by Robert C. Martin.
```
Comment (ntache-81) [inline contributions.md:39]: I thought a couple examples of how we interpret this section would be helpful for contributors. OK?

File: contributions.md [hunk 6]
```diff
@@ -15,22 +15,34 @@ We are looking for contributions in the following areas:
...
+
+See the relevant section in [other practice pages](/practices/) for more examples.
 
 ## Adjacent Capabilities
 
-2-4 other DORA Capabilities that are related either because they're similar or because you need one to accomplish the other. Each capability should have a title and description. The title should be an already existing DORA Capability.
+The final section lists 2-4 other DORA Capabilities that are related to the Practice, either because they're similar or because you need one to accomplish the other. Each Capability should have a title (styled as a subhead) and brief (2-4 sentences) description. The title should be an  existing, linked DORA Capability from the repository.
```
Comment (ntache-81) [inline contributions.md:45]: Some practices (run pair programming and reduce coupling) have only ONE adjacent capability. Should those be revisited, or should this description be more flexible to allow for just one? 
e.g., ("The final section lists 1-4 other DORA Capabilities that are related to the Practice...")

File: contributions.md [hunk 7]
```diff
@@ -15,22 +15,34 @@ We are looking for contributions in the following areas:
...
 
-If you chose to contribute a new practice try to follow our pre-established structure:
+If you chose to contribute a new practice, please try to follow our pre-established structure:
 
 ```markdown
-# Title Of Practice
+# Title of Practice
 
-A 2-3 sentence description of the practice followed by a sentence or two describing the main benefits of adopting this practice. Try to keep it brief.
+A 2-3 sentence description of the practice, followed by a sentence or two describing the main benefits of adopting this practice. Try to keep the text brief, relatable, and motivational.
```
Comment (ntache-81) [inline contributions.md:23]: Would you like to also clarify expectations/style regarding the level of technical detail in these Practices? If so, we can add a line like this: "Our audience wants a quick read. The writing style should be conversational, not academic; the details should be higher level (think overview or summary), not too technical."

File: contributions.md (hunk 3 above)
Comment (ntache-81) [reply contributions.md:31]: **Gaining Traction** -- Nicole will edit to reflect this new title.

File: contributions.md (hunk 6 above)
Comment (dcmoore) [reply contributions.md:45]: we can update to only have 1 in the case where that makes sense

File: contributions.md (hunk 7 above)
Comment (dcmoore) [reply contributions.md:23]: yes please. The text you have works great.

**Pull request merged**